src/components/learner-credit-management/AssignmentRowActionTableCell.jsx

```diff
@@ -1,45 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {
-  Icon,
-  IconButton,
-  OverlayTrigger,
-  Stack,
-  Tooltip,
+  Icon, IconButton, OverlayTrigger, Stack, Tooltip,
 } from '@edx/paragon';
 import { Mail, DoNotDisturbOn } from '@edx/paragon/icons';
 
 const AssignmentRowActionTableCell = ({ row }) => {
-  const cancelButtonMarginLeft = row.original.state === 'allocated' ? 'ml-2.5' : 'ml-auto';
+  const learnerStateBool = row.original.learnerState === 'waiting';
+  const cancelButtonMarginLeft = learnerStateBool ? 'ml-2.5' : 'ml-auto';
   return (
-    <div className="d-flex">
-      {row.original.state === 'allocated' && (
-        <>
-          <OverlayTrigger
-            key="Remind"
-            placement="top"
-            overlay={<Tooltip variant="light" id="tooltip-remind">Remind learner</Tooltip>}
-          >
-            <IconButton
-              className="ml-auto mr-0"
-              src={Mail}
-              iconAs={Icon}
-              alt="Remind learner"
+    <Stack direction="horizontal" gap="1">
+      {learnerStateBool && (
+      <OverlayTrigger
+        key="Remind"
+        placement="top"
+        overlay={<Tooltip variant="light" id="tooltip-remind">Remind learner</Tooltip>}
+      >
+        <IconButton
+          className="ml-auto mr-0"
+          src={Mail}
+          iconAs={Icon}
+          alt="Remind learner"
               // eslint-disable-next-line no-console
-              onClick={() => console.log(`Reminding ${row.original.uuid}`)}
-              data-testid={`remind-assignment-${row.original.uuid}`}
-            />
-          </OverlayTrigger>
-          <Stack direction="horizontal" gap={1} />
-        </>
+          onClick={() => console.log(`Reminding ${row.original.uuid}`)}
+          data-testid={`remind-assignment-${row.original.uuid}`}
+        />
+      </OverlayTrigger>
       )}
       <OverlayTrigger
         key="Cancel"
         placement="top"
         overlay={<Tooltip variant="light" id="tooltip-cancel">Cancel assignment</Tooltip>}
```

Comment:
Similar comment here about the `Tooltip` variant. I believe we should use the default variant here such that the tooltip's background is dark vs. white, for parity with the subscriptions table:

<img width="1211" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/4c53da46-d3b1-47d0-89ad-f6cf8c5945b0">

The tooltip background for "Cancel assignment" in the Figma is the danger variant but this isn't a supported variant by Paragon's `Tooltip`; it only supports the default variant (dark) or light.